pygmt/src/plot.py

```diff
@@ -247,7 +247,7 @@ def plot(  # noqa: PLR0912
             kwargs["S"] = "s0.2c"
         elif kind == "file" and str(data).endswith(".gmt"):  # OGR_GMT file
             try:
-                with Path(which(data)).open() as file:
+                with Path(which(data)).open(encoding="utf-8") as file:
```

Comment:
This assumes that `*.gmt` files are always encoded in UTF-8, is this the case, or should we choose `encoding="locale"` instead?

Reply:
`*.gmt` can be encoded in any encoding, so there is no way to make it work in all cases. 

> To be clear, we're only trying to parse the vector geometry type (Multipoint/Point) from the *.gmt file, not read the whole file, so maybe ok to assume that the header lines are utf-8 compatible?

It usually works, but will fail if the file uses a different encoding and contains comments on the first line.

Actually, we're assuming that the `@G` record is always on the first line but that's not always true.

Reply:
`ogr2ogr` also produces OGR_GMT files with `# @VGMT1.0` followed by `@GP`, so it should be fine, although users can manually putting `@GP` at the second line and even add any comments at the begin of the file.

Anyway, that's just rare cases and it's safe to ignore them.